js/playlist.js

```diff
@@ -187,6 +187,21 @@ function(ytData,ytIframe,template,$) {
         savePlaylist();
     }
 
+    var clearPlaylist = function(){ 
+      playlist = [];
+      total_duration = 0;
+      //removes all the videos
+      $('#playlist li').remove();
+      //reset playlis duration
+      $('#totalDuration').text("00:00:00");
+      //save the playlist
+      savePlaylist();
+      //reset duration
+      saveDuration();
+      //change title
+      document.title= "Youtube Fast Playlist";
+    }
```

Comment:
3 things:

 - Rest of the code uses 4 space indents, please maintain that
 - You don't need to comment every line! The function `savePlaylist()` is descriptive enough in itself, so you don't need the extra comment. Ideally you should simply explain what the function does at the top of the function declare, and if it's very unclear add hints in the function body.
 - Everything works great right now, except that the playing video doesn't stop. Add `ytIframe.player.stopVideo();` to stop the video too.